receiver/flinkmetricsreceiver/metadata.yaml

```diff
@@ -0,0 +1,522 @@
+name: flinkmetricsreceiver
+
+resource_attributes:
+  host:
+    description: The host, used to distinguish different jobmanager metrics.
+    type: string
+  taskmanager_id:
+    description: The taskmanager ID, used to distinguish taskmanager metrics.
+    type: string
+  job_name:
+    description: The job name, used to distinguish job metrics.
+    type: string
+  task_name:
+    description: The task name, used to distinguish tasks metrics.
+    type: string
+  subtask_index:
+    description: The subtask index, used to distinguish operator metrics.
+    type: string
+
+attributes:
+  operator_name:
+    description: The operator name, used to distinguish operator metrics.
+    type: string
+  garbage_collector_name:
+    description: The garbage collector name.
+    type: string
+  checkpoint:
+    description: The amount of checkpoints of a given type. 
+    type: string
+    enum: [ in_progress, completed, failed ]
+  record:
+    description: The amount of records of a given type.
+    type: string
+    enum: [ in, out, late_records_dropped ]
+
+metrics:
+  flinkmetrics.jobmanager.status.jvm.cpu.load:
+    enabled: true
+    description: The recent CPU usage of the JVM.
+    unit: 1
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.jvm.cpu.load:
+    enabled: true
+    description: The recent CPU usage of the JVM.
+    unit: 1
+    gauge:
+      value_type: double
+      input_type: string
+    attributes: []
+  flinkmetrics.jobmanager.status.jvm.cpu.time:
+    enabled: true
+    description: The CPU time used by the JVM.
+    unit: ns
+    sum:
+      monotonic: true
+      aggregation: cumulative
+      value_type: int
+      input_type: string
+    attributes: []
+  flinkmetrics.taskmanager.status.jvm.cpu.time:
+    enabled: true
+    description: The CPU time used by the JVM.
```

Comment:
Same description. What's the difference?

Reply:
So there is one jobmanager but there can be many taskmanagers. So only a host is needed for a jobmanager, whereas a host and taskmanagerID is needed to uniquely distinguish metrics. 

In other comment, I propose adding a `flink.scope` resource attribute, which will create single metric. That way a metric can be easily distinguish by looking at a the `flink.scope` value instead of seeing that there is no `flink.taskmanager.id` and therefor has to be a jobmanager metric.

Reply:
I think it would be reasonable to have a resource attribute along the lines of what you've proposed, if only because it may be very difficult to apply processors otherwise.

As to the specific name, we need to stay away from `scope`, as this is used elsewhere in the data model. I think `flink.resource.type` is appropriate.